niceman/distributions/conda.py

```diff
@@ -27,6 +27,37 @@
 import logging
 lgr = logging.getLogger('niceman.distributions.conda')
 
+# Provides the conda platform mapping from comda/models/enum.py
+# Note that these are python prefixes (both 'linux2' and 'linux' from python
+# map to 'linux' in conda.
+python_to_conda_platform_map = {
```

Comment:
Since this is only used in the function below, why not make it local to that function?

Reply:
That was an artifact of the order in which I wrote it.  Originally I just had the dict, but I realized that the function made the dict much more useful as I was writing the test....  I'll combine it.